icons/headphone-off.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "../icon.schema.json",
+    "contributors": [
+      "colebemis",
+      "csandman",
+      "ericfennis",
+      "jguddas",
+      "Need-an-AwP"
+    ],
+    "tags": [
+      "music",
+      "audio",
+      "sound",
+      "mute",
+      "off"
+    ],
+    "categories": [
+      "multimedia",
+      "connectivity",
+      "devices",
+      "files",
```

Comment:
Not sure if `files` is the correct category for this one.

Reply:
this is written in imitation of the original headphone icon meta file
`"categories": [
    "multimedia",
    "connectivity",
    "devices",
    "files",
    "gaming"
  ]`
i dont know if the "files" tag is correct here🙄

Reply:
> this is written in imitation of the original headphone icon meta file `"categories": [ "multimedia", "connectivity", "devices", "files", "gaming" ]` i dont know if the "files" tag is correct here🙄

`headphone` could represent an audio file. `headphone-off`... not so much. 😄

Reply:
what about jjust copy the categories from mic-off's meta info?